foundations/tying_it_all_together/conclusion.md

```diff
@@ -1,43 +1,44 @@
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects, and becoming a great developer. There is no right answer here on what is the next best path. Learning multiple languages can help you become a well rounded developer, but sticking with one could make you more of an expert in the language itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
+You've come a long way but there is so much more cool stuff that you'll learn to do. Now is the time to decide which path you want to take going forward. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects, and becoming a great developer. There is no right answer here on what is the next best path. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
 
-Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, building an engineering mindset, working on complex problems and codebases, communicating effectively with other developers, and **much** more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
+Learning how to program is **much** more than just learning a language. It's about learning how to break down your problems into achievable steps, research and solve difficult problems, communicating effectively with other developers, and much more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
 
 Arguments about learning the languages used in your area may sound convincing, but because a _good_ developer can transfer skills easily, what you learn isn't going to massively change outcomes of getting a job. Though, if you cannot shake that fear, do whatever you need, even if that includes researching your area to see what jobs are available locally (don't forget about remote jobs though). A good course is far more important than the language you learn first. 
 
 To help you make a decision, here are some perks of each technology:
 
 #### Ruby and Ruby On Rails: 
-* Opinionated 
-  - You don't have to make decisions on the structure of your projects. This helps when moving into projects built by others.
```

Comment:
Perhaps the verbiage could be improved here, but specifically with Rails, this is critical and not worth outright deleting.

Reply:
I agree with your next comment about swapping Ruby for Rails, and I think that should cover this one as well.